.github/workflows/ci_cron_weekly.yml

```diff
@@ -74,3 +74,23 @@ jobs:
         python -m pip install tox
     - name: Test with tox
       run: tox -e py310-test-romandeps
+
+  ci_cron_tests_stable_strauss:
+    name: Python 3.10 with stable versions of dependencies and Strauss
+    runs-on: ubuntu-latest
+    if: (github.repository == 'spacetelescope/jdaviz' && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Extra CI')))
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+      with:
+        fetch-depth: 0
+    - name: Set up python
+      uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b  # v5.3.0
+      with:
+        python-version: '3.10'
+    - name: Install base dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox
+    - name: Test with tox
+      run: tox -e py310-test-straussdeps
```

Comment:
@pllim Please let me know if I did this correctly, I haven't done it before so any advice would be appreciated!

Reply:
Looks like it runs now. Successfully or not is another matter. Good luck!